standalone/WebSocketExportToOrg.py

```diff
@@ -0,0 +1,142 @@
+pathToOrgFile = "/path/to/file.org.
+printSummary = True
+base = PyOrgMode.OrgDataStructure()
+channelsList = extWebSocket.getStorage().getTable().getChannelItems()
+
+# Loop through the Channel List
+for channel in channelsList:
+  hr = channel.getHandshakeReference()
+  newChannelElement = PyOrgMode.OrgNode.Element()
+  newChannelElement.level = 1
+  newChannelElement.heading = "(#"+str(channel.id)+"): "
+  if(hr != None):
+    handshakeMessage = hr.getHttpMessage()    
+    newChannelElement.heading =  newChannelElement.heading + handshakeMessage.getRequestHeader().getHostName() + "\n"
+    #Print Summary
+    if(printSummary):
+      newSummary = PyOrgMode.OrgNode.Element() 
+      newSummary.heading = "Summary"
+      newSummary.level = 2
+      newSummary.append("\t - Host: " + handshakeMessage.getRequestHeader().getHostName() + "\n")
+      newSummary.append("\t - URI: " + handshakeMessage.getRequestHeader().getMethod() + " " + handshakeMessage.getRequestHeader().getURI().toString() + "\n")
+      newSummary.append("\t - Port: " + str(handshakeMessage.getRequestHeader().getURI().getPort()) + "\n")
+      newSummary.append("\t - Websocket Port: " + str(channel.port) +"\n")
+      newSummary.append("\t - Websocket Url: " + str(channel.url) + "\n")
+      newSummary.append("\t - Messages Opcodes: \n")
+      newSummary.append("\t - Outgoing Messages format: \n")
+      newSummary.append("\t - Incoming Messages format: \n")
+      newSummary.append("\t - Input Messages encryption: \n")
+      newSummary.append("\t - Output Messages encryption: \n")
+      newSummary.append("\t - Ping/Pong: \n")
+      newSummary.append("\t - Close message: Opcode= , Text= \n")
+      newChannelElement.append_clean(newSummary);
+    
+    # Print handshake request
+    newHandShake = PyOrgMode.OrgNode.Element()
+    newHandShake.heading = "Switching Protocol"
+    newHandShake.level = 2
+
+    # Print Request Header 
+    newHandShakeRequest = PyOrgMode.OrgNode.Element()
+    newHandShakeRequest.heading = "Handshake Request"
+    newHandShakeRequest.append("\t#+BEGIN_SRC html \n" + handshakeMessage.getRequestHeader().toString() + "\t#+END_SRC \n") 
+    newHandShake.append_clean(newHandShakeRequest)
+
+    # Print Response Header 
+    newHandShakeResponse = PyOrgMode.OrgNode.Element()
+    newHandShakeResponse.heading = "Handshake Response"
+    newHandShakeResponse.append("\t#+BEGIN_SRC html \n" + handshakeMessage.getResponseHeader().toString() + "\t#+END_SRC \n") 
+    newHandShake.append_clean(newHandShakeResponse)
+
+    newChannelElement.append_clean(newHandShake)
+    
+  print("Processing Channel ID: #" + str(channel.id))
+  
+  #Get Messages  
+  messageCriteria = WebSocketMessageDTO(channel)
+
+  #messagesList = extWebSocket.getWebsocketMessages(messageCriteria,None, None,None,0,0,10000) #Take a look at https://github.com/zaproxy/zap-extensions/pull/15065
```

Comment:
There's a typo in the pull request number, but better restore the old method instead, to avoid breaking the script.